### PR TITLE
Emails prefs page rejigery

### PR DESCRIPTION
**Pull request by Calum-Campbell**
## What does this change?
- Settings title -> Emails settings
- "Your currently receiving emails to this email address: XX " In settings tab
- Emails tab -> Emails & marketing
- Wider tabs in desktop view to accommodate longer wording of Emails & marketing

## What is the value of this and can you measure success?
- More clarity

## Screenshots
image
----------------------------------------------------------------------------

image

## Tested in CODE?
No

@katyabeer

### CODE REVIEW

**Comment by jfsoul** [comment]
Should the content of the tab (e.g. newsletter accordion) be horizontally aligned with the end of the last tab? It appears to be at the moment - but this PR changes that.

File: identity/app/views/profile/emailPrefs.scala.html [hunk 1]
```diff
@@ -43,30 +44,38 @@ <h2 class="form__heading">Your newsletters</h2>
...
                         <p>
                             <a href="@buildIdentityUrl("account/edit")" class="u-underline" data-link-name="identity : email : update">
-                                Update your email address
+                                Change email address via Account & profile tab
                             </a>
                         </p>
                     </li>
+                    <li>
+                        <a href="#identity-modal--newsletterFormat" class="u-underline js-email-subscription__formatFieldsetToggle" data-link-name="identity : email : format">
+                            Change email format (HTML/txt)
```
Comment (jfsoul) [inline identity/app/views/profile/emailPrefs.scala.html]: can we call this `text` instead of `txt`?

**Comment by katyabeer** [comment]
Agree with @jfsoul. Additionally (and sorry for only realising it now) it's best to group 'unsubscribe from all newsletters' with the actual newsletters (see attached).

Lastly, let's slightly tweak formatting of the email settings section: Space the text with the associated email address out from the text links below. Let's also lose the colon between the sentence and the email address. Add the full stop at the end of that sentence. Again, see attached.

Otherwise, it's looking good :)

screen shot 2018-05-02 at 15 07 56

**Comment by Calum-Campbell** [comment]
Closing in favour of https://github.com/guardian/frontend/pull/19612

**Pull request closed without merging**